fischkapp/src/components/Header.tsx

```diff
@@ -0,0 +1,14 @@
+import { StyledHeader } from "./styles/Header.styled";
+export const Header = () => {
+  return (
+    <StyledHeader>
+      <div className="header__logo">
```

Comment:
**nitpick:** Why styling by class names when using styled components? It's not a good practice - all styled elements should be created with styled-components. You will avoid such deep nesting